src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
+
+    List<Driver> getAll();
+
+    Driver update(Driver driver);
+
+    boolean delete(Long id);
+}
```

Comment:
Do we need an empty line at the end of the file

Reply:
It's already there. Just GitHub don't show it)